### ISSUE TITLE
FR: `jj resolve` should recommend `jj chmod` when hitting an executable bit conflict 

### ISSUE THREAD

**Issue by glencbz** [martinvonz/jj#2420]
Currently, `jj resolve` outputs something like:

```
Error: Failed to resolve conflicts: Only conflicts that involve normal files (not symlinks, not executable, etc.) are supported. Conflict summary for "foobar":
Conflict:
  Removing executable file with id e80f6f12e211b26ea895ee714a42792fc2629c9a
  Adding executable file with id 4aeaee0c9c39937dc96e3f67f3c597a9803938ef
  Adding executable file with id 2e8e7acb76c93096f23358d06d0b9fb9c6124c22
```

Which hints that the solution has to do with the executable bit, but not how to change it in a way that `jj` can understand. The first tool users will probably reach for is `chmod`, but that doesn't work (because it doesn't touch the mtime..?), making this problem extra-frustrating.

The correct answer is to `jj chmod` the file, but the discoverability feels low. We should call out  `jj chmod`, and ideally, give the appropriate `<mode>` argument that makes `jj`.

**Comment by martinvonz** [comment]
Makes sense. There's also #1279 about not even considering most cases conflicts. In your case above, for example, the result should be an executable file (it was executable before and it was still executable afterwards on both sides of the conflict).

**Comment by ilyagr** [comment]
This should be superseded by the fix to #1279. 🎉 

Feel free to reopen if I missed something.
